dds/DCPS/LinuxNetworkConfigMonitor.cpp

```diff
@@ -50,7 +50,7 @@ bool LinuxNetworkConfigMonitor::open()
   struct {
     nlmsghdr header;
     rtgenmsg msg;
-  } request = {};
+  } request;
```

Comment:
does this need a memset or similar?

Reply:
Maybe? That's part of the reason I asked for Justin's review. At least for my system's `nlmsghdr`:

```c
struct nlmsghdr {
	__u32		nlmsg_len;	/* Length of message including header */
	__u16		nlmsg_type;	/* Message content */
	__u16		nlmsg_flags;	/* Additional flags */
	__u32		nlmsg_seq;	/* Sequence number */
	__u32		nlmsg_pid;	/* Sending process port ID */
};
```

We're setting everything but `nlmsg_seq`. The [man page](https://man7.org/linux/man-pages/man7/netlink.7.html) does seem to suggest that we should set it and track it.

My system's `rtgenmsg` just has `rtgen_family` that we're setting.

Reply:
`nlmsg_seq` seems to be a distinct issue.
For this change in particular I think the safest thing to do is to memset to 0 the number of bytes based on sizeof.  That makes it compatible with different system headers.  The man page is less than clear on what it expects.

Reply:
Yes, that's fine with me.

Reply:
I agree with the `memset` recommendation.